esque/config/__init__.py

```diff
@@ -198,40 +185,40 @@ def _get_confluent_ssl_config(self) -> Dict[str, str]:
             return {}
         rdk_conf = {}
         if self.current_context_dict.get("ssl_cafile"):
```

Comment:
if self.ssl_params.get("cafile"):

Reply:
sorry, I would have "suggested changes" but github was doing something weird with the indenting, so I just commented